resources/eventing/charts/controller/values.yaml

```diff
@@ -0,0 +1,18 @@
+# override name to avoid collision with kantive eventing resources
```

Comment:
kantive -> knative

Reply:
nit: I am wondering where you expect a collision? Knative eventing-controller is installed into knative-eventing namespace whereas this controller is installed into kyma-system.